mesheryctl/cmd/perf.go

```diff
@@ -127,9 +127,9 @@ func init() {
 	perfCmd.Flags().StringVar(&testName, "name", StringWithCharset(8), "DESCRIPTION")
 	perfCmd.Flags().StringVar(&testMesh, "mesh", "", "DESCRIPTION")
 	// perfCmd.Flags().StringVar(&testFile, "file", "", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&qps, "qps", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&parallelRequests, "parallel-requests", "1", "DESCRIPTION")
-	perfCmd.Flags().StringVar(&testDuration, "duration", "10s", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&qps, "qps", "0", "DESCRIPTION")
+	perfCmd.Flags().StringVar(&parallelRequests, "Concurrent requests", "1", "DESCRIPTION")
```

Comment:
shouldn't this be `concurrent-requests`? I'd also suggest you rename the var name to actually match (e.g. `concurrentRequests`)

Reply:
Great feedback, @peterj.

Thanks for taking this on, @ashiscs!